src/web/crate_details.rs

```diff
@@ -271,10 +275,20 @@ impl CrateDetails {
         Some(crate_details)
     }
 
-    /// Returns the version of the latest release of this crate.
+    /// Returns the version of the latest non-yanked release of this crate (or latest yanked if
+    /// they are all yanked).
     pub fn latest_version(&self) -> &str {
-        // releases will always contain at least one element
-        &self.releases[0].version
+        let release = self
+            .releases
+            .iter()
+            .find(|release| !release.yanked)
+            .unwrap_or_else(|| {
+                self.releases
+                    .iter()
+                    .next()
+                    .expect("there is always at least one release")
```

Comment:
Does this work without running into borrowck issues?
```suggestion
                &self.releases[0]
```

Reply:
It should